### PR TITLE
 ✨ update the landing page to match the design

### PR DESCRIPTION
**Pull request by wei-lee**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-7943
## What
Implement the landing page as in the design [here](https://redhat.invisionapp.com/share/Z2O2VWUU9J8#/screens/320204122)

## Why

See JIRA

## How

1. show a landing page if there is no mobile apps created
2. show the mobile apps card view if there are apps.

## Verification Steps

Run it against an existing cluster.

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Additional Notes

There are still a few things need to be done, I will create JIRAs and link them here.

### CODE REVIEW

File: ui/src/components/App.js [hunk 1]
```diff
@@ -10,6 +11,7 @@ import './App.css';
 const App = () => (
   <Router>
     <div className="App">
+      <HEADER />
```
Comment (wojta) [inline ui/src/components/App.js]: Why uppercase?

File: ui/src/components/overview/MobileClientCardView.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    this.setState({filter: filterValue});
+  }
+
+  updateCurrentValue(event) {
+    this.setState({ currentValue: event.target.value });
+  }
+
+  onValueKeyPress(keyEvent) {
+    const { currentValue } = this.state;
+    if (keyEvent.key === 'Enter' && currentValue && currentValue.length > 0) {
```
Comment (wojta) [inline ui/src/components/overview/MobileClientCardView.js]: shouldn't be there a way to do the filtering without keyboard?

**Comment by psturc** [comment]
@wojta for filtering, there is a [separate task](https://issues.jboss.org/browse/AEROGEAR-7994) I'm currently working on

File: ui/src/components/App.js (hunk 1 above)
Comment (wei-lee) [reply ui/src/components/App.js]: no good reason, will change it

File: ui/src/components/overview/MobileClientCardView.js (hunk 2 above)
Comment (wei-lee) [reply ui/src/components/overview/MobileClientCardView.js]: how?

**Pull request merged**